[IMP] pos: update the register customers section (#4662)
odoo/documentation: content/applications/sales/point_of_sale.rst

:show-content:

> :show-toc:

Reviewer: Do you delete this because it looks ugly?
If so, leave it there.

Don't break the normal navigation of the whole documentation just on this page, for now, although I agree we could reconsider if displaying the toc tree like it is done now is still relevant.

Author: I've chosen the "cards" suggestion, it looks fine :)

:guilabel:`Resume` if the session was already open.

.. note::

>    :doc:`Multiple users <point_of_sale/employee_login>` can be connected to the same session at the
>    same time. However, the session can only be opened once on the same browser.

Reviewer 2: ```suggestion
   :doc:`Multiple users <point_of_sale/employee_login>` can be logged into the same session at the
   same time. However, the session can only be opened once on the same browser.
```

.. _pos/customers:

Set customers
=============

You can create and set customers from an :ref:`open POS session <pos/session-start>`. Registering
your customer is necessary to :doc:`collect their loyalty points and grant them rewards
<point_of_sale/pricing/loyalty>`, automatically apply the :doc:`attributed pricelist
<point_of_sale/pricing/pricelists>`, or :ref:`generate and print an invoice <receipts/invoices>`.

To access the list of customers, click :guilabel:`Customer` on the POS interface. Then, select a
customer or create a new one by clicking :guilabel:`Create`, completing the form, and saving.

> Lastly, click :guilabel:`Set Customer` to set the selected customer or your newly created client.

Reviewer 2: ```suggestion
Lastly, click :guilabel:`Set Customer` to set the selected customer.
```
If feel like the "newly created client" is included in the "selected customer", since they're selected in bioth cases? (if you still want to keep "newly created client", I would write "the newly created ..." instead of "your ..."
